pubspec.yaml

```diff
@@ -1,11 +1,10 @@
-name: uikit
+name: dash_kit_uikit
 description: UI Kit for any new Flutter project.
```

Comment:
The dash component for creation of your own UI component library

Reply:
Fixed